mavros/src/plugins/local_position.cpp

```diff
@@ -68,19 +71,32 @@ class LocalPositionPlugin : public MavRosPlugin {
 
 	void handle_local_position_ned(const mavlink_message_t *msg, uint8_t sysid, uint8_t compid) {
 		mavlink_local_position_ned_t pos_ned;
+		Eigen::Vector3d angular_velocity;
 		mavlink_msg_local_position_ned_decode(msg, &pos_ned);
 
 		auto position = UAS::transform_frame_ned_enu(Eigen::Vector3d(pos_ned.x, pos_ned.y, pos_ned.z));
+		auto velocity = UAS::transform_frame_ned_enu(Eigen::Vector3d(pos_ned.vx, pos_ned.vy, pos_ned.vz));
 		auto orientation = uas->get_attitude_orientation();
+		auto imu_data = uas->get_attitude_imu();
+		tf::vectorMsgToEigen(imu_data->angular_velocity, angular_velocity);
+		angular_velocity = UAS::transform_frame_ned_enu(angular_velocity);
```

Comment:
Why?! IMU data is already converted. You may simply copy geometry_msgs::Vector3.

Also because you get whole imu message there no reason for getting orientation, it is inside sensor_msgs::IMU.